dir.props

```diff
@@ -152,6 +152,13 @@
 
     <ProjectUrl>https://dot.net</ProjectUrl>
 
+    <!-- Central place to set the versions of all nuget packages produced in the repo -->
+    <Version Condition="'$(Version)' == ''">1.1.0</Version>
```

Comment:
You might want to double check that no-one else is using the "Version" property.  If they are you could condition this to only be set when `'$(MSBuildProjectExtension)' == '.pkgproj'`